assume_various_securityaudit_roles.tf

```diff
@@ -0,0 +1,51 @@
+# IAM policy document that allows assumption of the SecurityAudit role
+# in the various accounts.
+data "aws_iam_policy_document" "assume_various_securityaudit_roles_doc" {
+  statement {
+    effect = "Allow"
+
+    actions = ["sts:AssumeRole"]
+
+    resources = [
+      "arn:aws:iam::${local.account_ids["audit"]}:role/${var.various_securityaudit_role_name}",
+      "arn:aws:iam::${local.account_ids["dns"]}:role/${var.various_securityaudit_role_name}",
+      "arn:aws:iam::${local.account_ids["images"]}:role/${var.various_securityaudit_role_name}",
+      "arn:aws:iam::${local.account_ids["logarchive"]}:role/${var.various_securityaudit_role_name}",
+      "arn:aws:iam::${local.account_ids["sharedservices"]}:role/${var.various_securityaudit_role_name}",
+      "arn:aws:iam::${local.account_ids["terraform"]}:role/${var.various_securityaudit_role_name}"
```

Comment:
Why not the users and master accounts?  If they are being intentionally left out then there should be a comment explaining why.

(I see below that the users account is handled differently, but there should still be a comment here.)